### PR TITLE
'showOptionTooltips' option introduced

### PR DESCRIPTION
**Pull request by kebabmaster**
For your consideration:

Introduced a new **Slim Select** option - **showOptionTooltips**.

This option is used to active displaying the on-hover tooltips for select options. The tooltip text is equal to the option text content. 

Example:
**`new SlimSelect({`
`   select: '#showOptionTooltips',`
`   showOptionTooltips: true`
`})`**

The documentation was updated accordingly.

### CODE REVIEW

**Comment by kebabmaster** [comment]
Hi @brianvoe,

In the company I work for, we actually had a use-case for the option proposed. In a modal dialog where Slim Select was utilised, there was very little space left for it and we had to resort to using the tooltips (the wordbreak option was turned down due to its being quite unsightly).

Below is an example set of data (OIDs from the Solid REL6 MIB):
rouTemperature
rouTemperatureHigh
rouTemperatureLow
rouTempHighAlarm
rouTempLowAlarm
rouDcAlarm
rouDoorAlarm

In the described scenario, for the temperature-related OIDs, only the following part of the name would fit the option: "rouTemp". Due to the user base, which is moderately or extensively experienced RF engineers, such a way of displaying the data was deemed acceptable. So, if the user had to select some ROU temperature-related value, they would navigate to the "rouTemp" set of options and utilise the tooltip to help them find the required OID. 

Two extra points:
1. Obviously, the examples are a bit of exaggeration, simply for better representation (and because I only had the SOLID REL6 MIB handy). In reality, the OID length can go as far as 100 characters.
2. I wish I could describe the use-case in better details; however, my contract and the exclusivity agreements are preventing me from doing so.

To sum up, I believe that the proposed functionality would be helpful to the developers who might find themselves in a similar situation to mine.

**Comment by brianvoe** [comment]
Find with me ill get this merged it. If you are ever able to show me a screen shot that would be great.

**Comment by brianvoe** [comment]
merged v1.17.0
